GDJS/patches/@pixi+tilemap+3.2.2.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/node_modules/@pixi/tilemap/index.d.ts b/node_modules/@pixi/tilemap/index.d.ts
```

Comment:
maybe a dumb question, but why is this patch needed?

Reply:
There is no such things as a dumb question :) This is needed because we have stricter type checking rules as the pixi-tilemap project, so their type definition has something that our typescript configuration doesn't allow: redefining a method present on the parent class. Since the type definition is the same and inherited anyways, this doesn't change much, but without it typescript ignores the pixi-tilemap typings as invalid types with our current configuration.